doc/reference/kernel/other/float.rst

```diff
@@ -77,6 +77,19 @@ This means that any thread is allowed to access the floating point registers.
 The ARM kernel automatically detects that a given thread is using the floating
 point registers the first time the thread accesses them.
 
+It is recommended to pretag a thread that intends to use the FP registers,
```

Comment:
This would read a bit better
```suggestion
Pretag a thread that intends to use the FP registers by
```

Reply:
Will do that too, thanks.

Reply:
@dbkinder I just want to make sure that we don't give the impression that pre-tagging is (absolutely) required for FP to work. That's why I added the "recommended". I've changed to what you suggested, pls, ACK